docs/search/criteria_reference/symbolattribute_criterion.md

```diff
@@ -0,0 +1,22 @@
+---
+description: SymbolAttribute Criterion
+---
+
+# SymbolAttributeCriterion
+
+The `SymbolAttribute` Search Criterion searches for products by symbol attribute.
```

Comment:
Because the package is optional, I'd link to the main doc for the attribute from here (docs/pim/symbol_attribute_type.md) - so that people can find more information about it directly from this page when browsing the criteria reference.